app/presenters/tree_builder_services.rb

```diff
@@ -18,16 +20,12 @@ def set_locals_for_render
 
   # Get root nodes count/array for explorer tree
   def x_get_tree_roots(count_only, _options)
-    roots = rbac_filtered_objects(Service.roots)
-
-    # Preload the root service picture since it's called by TreeNodeBuilder#build
-    MiqPreloader.preload(roots, :picture)
-
-    count_only_or_objects(count_only, roots, "name")
-  end
-
-  def x_get_tree_service_kids(object, count_only)
-    objects = rbac_filtered_objects(object.direct_service_children.select(&:display).sort_by { |o| o.name.downcase })
-    count_only_or_objects(count_only, objects, 'name')
+    all_services = Rbac.filtered(Service.where(:display => true))
```

Comment:
I admit to not taking the time to look this up, but do we know what the difference is between `rbac_filtered_objects` and `Rbac.filtered`)?  I assume they are equivalent, but just want to make sure this wasn't something that is a `darga` specific method here.